src/Engine/ProtoAssociative/CodeGen.cs

```diff
@@ -2195,6 +2240,7 @@ private void SSAIdentList(AssociativeNode node, ref Stack<AssociativeNode> ssaSt
                     bnode.Optr = ProtoCore.DSASM.Operator.assign;
                     bnode.isSSAAssignment = true;
                     bnode.isSSAPointerAssignment = true;
+                    bnode.IsFirstIdentListNode = true;
```

Comment:
Would you think about rewrite `SSAIdentList` with visitor pattern? :-) (ref: `AstReplacer`)